src/EventListener/ReplaceInsertTagsListener.php

```diff
@@ -22,9 +22,6 @@ public function __construct(MapManager $mapManager)
         $this->mapManager = $mapManager;
     }
 
-    /**
-     * @Hook("replaceInsertTags")
```

Comment:
was not used because it's registered in config.php and annotations are not supported with contao 4.4